osi_trafficsign.proto

```diff
@@ -347,90 +347,144 @@ message TrafficSign
+                // As symbolic road marking
+                // \c RoadMarking::Classification::TYPE_SYMBOLIC_TRAFFIC_SIGN
+                // (StVO 294)
+                //
+                // See e.g.: https://www.dvr.de/bilder/stvo/gt/294.png
+                //
                 TYPE_STOP = 17;
 
                 // Priority must be given to vehicles from the opposite direction
-                // (StVO 208). Or: #TYPE_PRIORITY_OVER_OPPOSITE_DIRECTION
+                // (StVO 208). Or: \c #TYPE_PRIORITY_OVER_OPPOSITE_DIRECTION
                 //
                 // See e.g.: https://www.dvr.de/bilder/stvo/gt/208.png
                 //
                 TYPE_PRIORITY_TO_OPPOSITE_DIRECTION = 18;
 
                 // Priority must be given to vehicles from the opposite direction
-                // (StVO 208 Upside down). Or: #TYPE_PRIORITY_TO_OPPOSITE_DIRECTION
+                // (StVO 208 Upside down). Or: \c #TYPE_PRIORITY_TO_OPPOSITE_DIRECTION
                 //
                 // See e.g.:
                 //
                 TYPE_PRIORITY_TO_OPPOSITE_DIRECTION_UPSIDE_DOWN = 19;
 
                 // Prescribed left turn
-                // (StVO 209.1). Right: #TYPE_PRESCRIBED_RIGHT_TURN
+                // (StVO 209.1). Right: \c #TYPE_PRESCRIBED_RIGHT_TURN
                 //
                 // See e.g.:
                 //
+                // As symbolic road marking
+                // \c RoadMarking::Classification::TYPE_SYMBOLIC_TRAFFIC_SIGN
+                // (StVO 297)
+                //
+                // See e.g.: https://www.dvr.de/bilder/stvo/gt/297.png
+                //
                 TYPE_PRESCRIBED_LEFT_TURN = 20;
 
                 // Prescribed right turn
-                // (StVO 209.2). Left: #TYPE_PRESCRIBED_LEFT_TURN
+                // (StVO 209.2). Left: \c #TYPE_PRESCRIBED_LEFT_TURN
                 //
                 // See e.g.: https://www.dvr.de/bilder/stvo/gt/209.png
                 //
+                // As symbolic road marking
+                // \c RoadMarking::Classification::TYPE_SYMBOLIC_TRAFFIC_SIGN
+                // (StVO 297)
+                //
+                // See e.g.: https://www.dvr.de/bilder/stvo/gt/297.png
+                //
                 TYPE_PRESCRIBED_RIGHT_TURN = 21;
 
                 // Prescribed driving straight ahead
                 // (StVO 209.3).
                 //
                 // See e.g.:
                 //
+                // As symbolic road marking
+                // \c RoadMarking::Classification::TYPE_SYMBOLIC_TRAFFIC_SIGN
+                // (StVO 297)
+                //
+                // See e.g.: https://www.dvr.de/bilder/stvo/gt/297.png
+                //
                 TYPE_PRESCRIBED_STRAIGHT_AHEAD = 22;
 
                 // Prescribed left way
-                // (StVO 211.1). Right: #TYPE_PRESCRIBED_RIGHT_WAY
-                //
+                // (StVO 211.1). Right: \c #TYPE_PRESCRIBED_RIGHT_WAY
+                // 
```

Comment:
Do we have a test checking for spaces before line break?

Reply:
No. I created unit tests mainly for errors. The space at the end of a line is more of an unwanted character and rather reduces the file size. A space at the end of a line doesn't cause an error, does it? Of course, it is very easy to add such an unit test. I suspect that every initial checkin will lead to an error with this unit test. Do we want to set up these high barriers for a check-in at GitHub? 

I personally support unit tests. Should we add this unit test?

Reply:
clang-format works for proto files, clang-check doesn't work for proto files (c,c++ works fine). We could use clang-format to reformat the files. Should we add active code-modification and github checkin via travis ci?

Reply:
yes, do we need a TravisCI user for that? The commit should be on the branch of the PR.

Reply:
Yes. We can use the same mechanism as for the auto-documentation. Drawback: We could end in an infinite loop 8-) of auto-check-ins.